Chicago95/info.json

```
@@ -0,0 +1,6 @@
+{
+    "author": "Grassmunk, AdrianoML, EMH-Mark-I",
```

```suggestion
    "author": "KAMI911",
    "original_author": "Grassmunk, AdrianoML, EMH-Mark-I",
```

This should be your GitHub username so that you can be notified on Issues that are created for it. You can put this current field as the original authors.